appveyor.yml

```diff
@@ -43,6 +43,7 @@ build_script:
   - travis-tool.sh r_install xts
   - travis-tool.sh r_install nanotime
   - travis-tool.sh r_install R.utils
+  - travis-tool.sh r_install yaml
```

Comment:
@jangorecki do we need to edit anything in to the GitLab CI for this? Or it's covered by the `knitr` dependency?

Reply:
it is enough to defined it in DESCRIPTION file